nh_eobs_ldap/models/res_users.py

```diff
@@ -0,0 +1,44 @@
+from openerp.osv import osv, fields
+from openerp.tools.translate import _
+from openerp import api
+
+
+class EOBSResUsers(osv.osv):
+    """ User class. A res.users record models an OpenERP user and is different
+        from an employee.
+
+        res.users class now inherits from res.partner. The partner model is
+        used to store the data related to the partner: lang, name, address,
+        avatar, ... The user model is now dedicated to technical data.
+    """
```

Comment:
Does this docstring apply to another override? If rewrite it so its specific to this one.

Reply:
👍